packages/block-library/src/image/image.js

```diff
@@ -690,8 +690,7 @@ export default function Image( {
 						) }
 						renderContent={ () => (
 							<TextControl
-								// TODO: Switch to `true` (40px size) if possible
-								__next40pxDefaultSize={ false }
+								__next40pxDefaultSize
```

Comment:
Kind of hard to get to this state. See testing instructions for #58998 if you're interested 😅

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/4745d768-40e8-4b52-8a42-c84e4d53886d" alt="Image block, content only mode, before" width="516">|<img src="https://github.com/user-attachments/assets/3dbc354a-f9a4-4213-b8e3-efed4fc133f6" alt="Image block, content only mode, after" width="516">|